inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/mvn/mvn_kernel_b_fs_yx_fsv16_imad.cpp

```diff
@@ -82,16 +82,33 @@ MVNKernelBase::DispatchData MVNKernel_b_fs_yx_fsv16_imad::SetDefault(const mvn_p
     return dispatchData;
 }
 
+Datatype MVNKernel_b_fs_yx_fsv16_imad::GetAccumulatorType(const mvn_params& params) const {
+    const auto& input_dt = params.inputs[0].GetDType();
+
+    switch (input_dt) {
+        case Datatype::F32:
+        case Datatype::F16:
+            return Datatype::F32;
+        case Datatype::INT8:
+        case Datatype::UINT8:
+            return Datatype::INT32;
+        default: return Datatype::F32;
```

Comment:
Isn't it necessary to evaluate performance impact from this change? Maybe we can make separate PR that may have potential performance impact..

Reply:
An unit test for this already exists. I missed it.